network/endpoint_windows.go

```diff
@@ -124,16 +142,154 @@ func (nw *network) newEndpointImpl(epInfo *EndpointInfo) (*endpoint, error) {
 	return ep, nil
 }
 
+// newEndpointImplHnsV2 creates a new endpoint in the network using HnsV2
+func (nw *network) newEndpointImplHnsV2(epInfo *EndpointInfo) (*endpoint, error) {
+	var vlanid int
+	if epInfo.Data != nil {
+		if _, ok := epInfo.Data[VlanIDKey]; ok {
```

Comment:
if vlanID, ok....

then you can use the vlanID.(int) I think (instead of accessing the value again)